nailgun/entities.py

```diff
@@ -8916,3 +8916,17 @@ def search(self, fields=None, query=None, filters=None):
         return super().search(
             fields=fields, query=query, filters=filters, path_fields={'user': self.user}
         )
+
+
+class NotificationRecipients(Entity, EntityReadMixin):
+    """A representation of /notification_recipients endpoint."""
+
+    def __init__(self, server_config=None, **kwargs):
+        self._fields = {
+            'notifications': entity_fields.ListField(),
+        }
+        self._meta = {
```

Comment:
Could you add a unit test for this entity class?

Reply:
Unit test added and passing:
```
pytest tests/test_entities.py -k NotificationRecipientsTestCase
============================================================================================================= test session starts ==============================================================================================================
...
collected 163 items / 162 deselected / 1 selected                                                                                                                                                                                              

tests/test_entities.py::NotificationRecipientsTestCase::test_to_json PASSED                                                                                                                                                              [100%]

====================================================================================================== 1 passed, 162 deselected in 0.20s =======================================================================================================
```